<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pidMenu</class>
 <widget class="QTabWidget" name="pidMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>715</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.532338000000000" starty="0.000000000000000" endx="0.526975000000000" endy="0.864000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>103</red>
          <green>103</green>
          <blue>103</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>MS Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>PIDC Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0.532338, y1:0, x2:0.526975, y2:0.864, stop:0 rgba(103, 103, 103, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 0, 0);
border-color: rgb(0, 0, 0);

</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="mainVals">
   <attribute name="title">
    <string>RDDPMC Page</string>
   </attribute>
   <widget class="QLabel" name="depth">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QLabel" name="roll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="pitch">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLabel" name="rollp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>79</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>P:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rolli">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QLabel" name="rolld">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>D:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roll_P">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="roll_I">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>81</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>1234</string>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="roll_D">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>81</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="depth_I">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>251</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="depth_D">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>251</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="depthp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>249</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>P:</string>
    </property>
   </widget>
   <widget class="QLabel" name="depthd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>D:</string>
    </property>
   </widget>
   <widget class="QLabel" name="depthi">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depth_P">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pitch_I">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>411</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pitch_D">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>411</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pitchp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>P:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pitchd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>D:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pitchi">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pitch_P">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 8);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PIDApply">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8-bit Operator+ 8</family>
      <pointsize>-1</pointsize>
      <weight>25</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
 font-size: 40px;
  font-weight: 200;
  letter-spacing: 1px;
  padding: 13px 50px 13px;
  outline: 0;
  border: 3px solid;
border-radius:15px;

	border-color: rgb(255, 255, 255);

  position: relative;
  background-color: rgba(0, 255, 0, 0);
}

QPushButton:Hover{
position :-5px 5px;
label.left : -5px;
}
QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.502, y2:0.517091, stop:0 rgba(13, 110, 8, 255), stop:1 rgba(36, 225, 42, 255));


}</string>
    </property>
    <property name="text">
     <string>APPLY</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="PIDLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>560</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.502, y2:0.517091, stop:0 rgba(36, 225, 42, 255), stop:1 rgba(13, 110, 8, 255));
  width: 100%;
  z-index: -1;
  position: relative;
  height: 100%;
  top: 7px;
  left: 7px;
border-radius:15px;


}
QLabel::Hover{
  top: 0px;
  left: 0px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>depth</zorder>
   <zorder>roll</zorder>
   <zorder>pitch</zorder>
   <zorder>rollp</zorder>
   <zorder>rolli</zorder>
   <zorder>rolld</zorder>
   <zorder>roll_P</zorder>
   <zorder>roll_I</zorder>
   <zorder>roll_D</zorder>
   <zorder>depth_I</zorder>
   <zorder>depth_D</zorder>
   <zorder>depthp</zorder>
   <zorder>depthd</zorder>
   <zorder>depthi</zorder>
   <zorder>depth_P</zorder>
   <zorder>pitch_I</zorder>
   <zorder>pitch_D</zorder>
   <zorder>pitchp</zorder>
   <zorder>pitchd</zorder>
   <zorder>pitchi</zorder>
   <zorder>pitch_P</zorder>
   <zorder>PIDLabel</zorder>
   <zorder>PIDApply</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
